bin-src/lib/getDependentStoryFiles.test.ts

```diff
@@ -470,6 +470,244 @@ describe('getDependentStoryFiles', () => {
+      },
+    ];
+    const ctx = getContext({
+      untraced: ['**/preview.js'],
+    });
+    const res = await getDependentStoryFiles(ctx, { modules }, statsPath, changedFiles);
+    expect(ctx.turboSnap.bailReason).toBeUndefined();
+    expect(res).toEqual({
+      './src/foo.stories.js': ['src/foo.stories.js'],
+    });
+  });
+
+  it('does not bail on changed wrapper component of untraced config file', async () => {
+    const changedFiles = ['src/utils.js', 'src/docs-wrapper.jsx'];
+    const modules = [
+      {
+        id: './path/to/storybook-config/preview.js-generated-config-entry.js',
+        name: './path/to/storybook-config/preview.js-generated-config-entry.js + 28 modules', // untraced
+        modules: [
+          { name: './path/to/storybook-config/preview.js-generated-config-entry.js' },
+          { name: './src/docs-wrapper.jsx' },
+        ],
+        reasons: [],
+      },
+      {
+        id: './src/docs-wrapper.jsx',
+        name: './src/docs-wrapper.jsx', // untraced
+        reasons: [{ moduleName: './src/docs-wrapper.stories.js' }],
+      },
+      {
+        id: './src/utils.js',
+        name: './src/utils.js', // changed
+        reasons: [{ moduleName: './src/foo.js' }],
+      },
+      {
+        id: './src/foo.js',
+        name: './src/foo.js',
+        reasons: [{ moduleName: './src/foo.stories.js' }],
+      },
+      {
+        id: './src/foo.stories.js',
+        name: './src/foo.stories.js',
+        reasons: [{ moduleName: CSF_GLOB }],
+      },
+      {
+        id: CSF_GLOB,
+        name: CSF_GLOB,
+        reasons: [{ moduleName: './path/to/storybook-config/generated-stories-entry.js' }],
+      },
+    ];
+    const ctx = getContext({
+      configDir: 'path/to/storybook-config',
+      untraced: [
+        '**/docs-wrapper.jsx',
+        '**/path/to/storybook-config/preview.js-generated-config-entry.js',
+      ],
+    });
+    const res = await getDependentStoryFiles(ctx, { modules }, statsPath, changedFiles);
+    expect(ctx.turboSnap.bailReason).toBeUndefined();
+    expect(res).toEqual({
+      './src/foo.stories.js': ['src/foo.stories.js'],
+    });
+  });
+
+  it('does not bail on changed wrapper component of untraced config file and untraced dependencies of a wrapper component', async () => {
+    const changedFiles = ['src/utils.js', 'src/packages/design-system/components/button.jsx'];
+    const modules = [
+      {
+        id: './path/to/storybook-config/preview.js-generated-config-entry.js',
+        name: './path/to/storybook-config/preview.js-generated-config-entry.js + 28 modules', // untraced
+        modules: [
+          { name: './path/to/storybook-config/preview.js-generated-config-entry.js' },
+          { name: './src/docs-wrapper.jsx' },
+        ],
+        reasons: [],
+      },
+      {
+        id: './src/docs-wrapper.jsx',
+        name: './src/docs-wrapper.jsx',
+        reasons: [{ moduleName: './src/packages/design-system/components/button.jsx' }],
```

Comment:
This is backwards I think? This is saying that `./src/packages/design-system/components/button.jsx` imports `./src/docs-wrapper.jsx` -- but if I understand the test correctly, I think you mean the opposite?